Wrappers/Python/cil/optimisation/operators/SymmetrisedGradientOperator.py

```diff
@@ -107,7 +125,8 @@ def direct(self, x, out=None):
             
                                                
     def adjoint(self, x, out=None):
-        
+        '''Returns the adjoint of the symmetrised gradient operator'''        
+       
```

Comment:
Please update to add docs on 'out', similar to in 'direct' above

Reply:
Thanks @lauramurgatroyd